src_c/mixer.c

```diff
@@ -343,20 +345,22 @@ _init(int freq, int size, int stereo, int chunk, char *devicename)
     if (!size) {
         size = request_size;
     }
-    if (!stereo) {
-        stereo = request_stereo;
+
+    if (!channels) {
+        channels = request_channels;
     }
+    channels = (channels >= 2) ? 2 : 1;
```

Comment:
I wonder if we should let `channels` to go through as a number more than 2? I guess none of the code is prepared for this really (sound array etc), and it would require testing. But SDL2 accepts more than 2.

https://wiki.libsdl.org/SDL_AudioSpec#Remarks
```
channels specifies the number of output channels. As of SDL 2.0, supported values are 1 (mono), 2 (stereo), 4 (quad), and 6 (5.1). 
```

Reply:
So, I guess this `channels >= 2` test should only be for SDL1.
For SDL2 it should check for 1, 2, 4, and 6.